soc/sifive/sifive_freedom/fu700/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_SIFIVE_FREEDOM_U700
+	bool
+	select SOC_FAMILY_SIFIVE_FREEDOM
+
+config SOC_SERIES
+	default "fu700" if SOC_SERIES_SIFIVE_FREEDOM_U700
```

Comment:
As above, name should match Kconfig

Reply:
Dropped the `f` from the start of the string